### ISSUE TITLE
Add debate name/purpose as metadata

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#55]
Each debate has a "title". This is relevant metadata to add in the corpus per debate.

**Comment by ninpnin** [comment]
Are these usually included as margin notes?
Parla Clarin example https://github.com/welfare-state-analytics/riksdagen-corpus/blob/dev/data/new-parlaclarin/prot-198081--43.xml#L243 
Associated image scan https://betalab.kb.se/prot-198081--43/prot_198081__43-003.jp2/_view

**Comment by MansMeg** [comment]
Im not sure that they are always included as margin notes, but in some debates they are.

**Comment by ninpnin** [comment]
Then I'm wondering if there's a heuristic for finding them. Maybe a number and this symbol § . Otherwise more annotation is needed.

**Comment by MansMeg** [comment]
Yes. I think we would need annotation fir this. Although for debates 1993- they should be included?
